guides/WRITING-DOCUMENTATION.md

```diff
@@ -15,176 +15,147 @@ For a component named "Foo", you would have the following:
 
 The expected format is:
-`[Top-level slug].[Subpage slug].[Page section slug]`
+`[Top-level slug].[Subpage slug].[Page section slug]` for example `Style guide: components.button.disabled`
 
-As you might be able to tell from the format, the docs site supports a maximum of 2 levels of nesting. Pages nested a 3rd level will be displayed in the body of their parent page.
+The docs site supports a maximum of 2 levels of nesting. Pages nested a 3rd level will be displayed in the body of their parent page.
 
-Pages generated from KSS comment blocks are ordered alphabetically, and page sections are displayed in the order in which they are in the CSS file.
+Pages generated from KSS comment blocks are ordered alphabetically, and page sections are displayed in the order in which they are in the SCSS file.
 
-### Guidance
+### Documentation and guidance
 
-To add content to the "Guidance" tab of a page, the page section's slug should begin with `guidance`. For example: `components.buttons.guidance`
+To add guidance content to the page, the page section's slug should be `guidance`. 
+For example: `components.buttons.guidance`
 
-To link to another documentation page, you can prefix the relative url with `{{root}}` to have the correct path added to the URL when the site is generated. For example: `{{root}}/components/button`
+Guidance can be written using [Markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) if you include `---` right after your opening section comments, `/*`. For example
 
-In general, the guidance comment block should be placed at the end of the CSS file and conform to the following format:
+To link to another documentation page in your guidance content use a relative url. **Example:** `/components/button`
+
+Documentation and guidance should be placed at the end of the SCSS file and conform to the following format:
 
-<details>
-<summary>View example</summary>
 
-```css
+```scss
 /*
 ---
 
-## When to use
-
-- List typical use cases for this component
-
-## When to consider alternatives
-
-- List commonly confused use cases where an alternative
-  component would be a better solution
-
 ## Guidance
 
-- List any design or content guidelines related to how
-  this component should be used
-
-## Accessibility
-
-- List any accessibility considerations like required
-  keyboard functionality, ARIA attributes, etc
-
-## Theming
-
-- List any Sass variables a developer could override
-  to customize the appearance of the component
-
-## Future research
-
-- Optionally list any suggestions for future research
-  that could mature and extend the component
-
-## Related patterns
-
-- List any related components or utility classes that
-  connect with the component, whether as a parent, sibling,
-  or child within the pattern's taxonomy.
-
-## Learn more
-
-- List any links that served as inspiration, references,
-  or research related to this component.
+Content here
```

Comment:
?

Reply:
I added documentation guidance below and pulled it out of the code block. We can add it back if you think that makes more sense. I was thinking it would be easier to understand as guidance to follow outside of the code block.

Reply:
Let's find out!